scripts/build/fiwarelab/build-fiwarelab-image.sh

```diff
@@ -104,27 +108,20 @@ MONGO_REPO=$(mktemp)
 cat > $MONGO_REPO <<EOF
 [mongodb]
 name=MongoDB Repository
-baseurl=http://downloads-distro.mongodb.org/repo/redhat/os/x86_64/
-gpgcheck=0
+baseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.2/x86_64/
+gpgcheck=1
 enabled=1
+gpgkey=https://www.mongodb.org/static/pgp/server-3.2.asc
 EOF
 sudo mv $MONGO_REPO /etc/yum.repos.d/mongodb.repo
 
-MONGO_VERSION=2.6.9
+MONGO_VERSION=3.2.6
 sudo yum install -y mongodb-org-$MONGO_VERSION \
                     mongodb-org-server-$MONGO_VERSION \
                     mongodb-org-shell-$MONGO_VERSION \
                     mongodb-org-mongos-$MONGO_VERSION \
                     mongodb-org-tools-$MONGO_VERSION
 
-# Default MongoDB configures set doesn't enables smallfiles by default. Enabling it ensures that MongoDB will 
```

Comment:
Given that WiredTiger is default storage in MongoDB 3.2 (according to https://docs.mongodb.com/manual/release-notes/3.2/#wiredtiger-as-default) and that smallfiles applies only to mmapv1 (see https://docs.mongodb.com/manual/reference/configuration-options/#storage-options) this is no longer needed.

NTC (just informative)